addons/mail/static/src/models/message/message.js

```diff
@@ -442,6 +442,23 @@ function factory(dependencies) {
                 this.originThread.isModeratedByCurrentPartner
             );
         }
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsSubjectEqualThreadName() {
+            if (!this.subject || !this.originThread || !this.originThread.name) {
+                return false;
+            }
+            const prefix = ['re:', 'fw:', 'fwd:'];
+            let cleanedSubject = this.subject.toLowerCase();
+            prefix.forEach((pref) => {
+                if (cleanedSubject.startsWith(pref)) {
+                    cleanedSubject = cleanedSubject.replace(pref, '');
+                }
+            });
```

Comment:
You need to apply the same clean up to the `originThread.name` .

Because imagine if the thread name is "re: test", and you get a subject that is "re: test" current code will say it's different when it's actually equal.

Reply:
Maybe an alternative solution to this problem is to check whether "cleanedSubject === originThread.name" at the start of each iteration, and immediately return true if that is the case.